### PR TITLE
Changed plugin name to tumblr theme garden

### PR DESCRIPTION
**Pull request by aaronjbaptiste**
### Summary

Updates project references from tumblr theme translator to tumblr theme garden

### Testing steps

1. Checkout branch and launch in Wordpress Studio
2. Check plugin is activated
3. Check you can activate a theme
4. Check you can customize the theme
5. Try swapping to a different theme

### Reviewing tips

🧁 Request a review to team CupcakeLabs to assign a random team member.

🔄 Feel free to ping the team name again to "re-roll" or add another team member.

🚀 For now there is no requirement on getting review approvals before merging.

### CODE REVIEW

**Review by roccotripaldi — APPROVED** [review]
This works for me! After this merges, I'm going to name a few of my files from theme-garden -> theme-browser to avoid confusion.

Also, I'll look into renaming the root folder's name. It's still called T3 when I clone the repo.

**Comment by aaronjbaptiste** [comment]
> This works for me! After this merges, I'm going to name a few of my files from theme-garden -> theme-browser to avoid confusion.
> 
> Also, I'll look into renaming the root folder's name. It's still called T3 when I clone the repo.

Thanks Rocco! There's a few references to T3 and tumblr3, including the repo name itself. I guess we need to decide on if we want T3 to be an internal name or do a mass rename! These changes should be enough to get the plugin submitted for now.

cc @tommusrhodus

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Tumblr Theme Translator (T3)
+# Tumblr Theme Garden
 
 > [!TIP]
 > [Take T3 for a spin in one-click right now on WordPress Playground!](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/Automattic/tumblr-theme-translator/refs/heads/trunk/blueprint.json)
```
Comment (MattPearce) [inline README.md:4]: It says "Take T3 for a spin", might need to update that? Also GitHub repo link needs updating

Comment (MattPearce) [reply README.md:4]: Oh, noticed this has actually been fixed in a later PR already

**Pull request merged**